test/TritonGPU/amd/amd-canonicalize-pointers.mlir

```diff
@@ -94,11 +94,24 @@ module attributes {"ttg.num-warps" = 4 : i32} {
 // -----
 
 
+// the original code is sketched below:
 //
-// This is the same as conversion3, but now the `arith.extsi` operations
-// disappeared and all the offsets are 32 bits.
+// %0 = t.program_id(aixs=0)
+// %1 = %0 * 1024
+// %2 = tl.arange(0, 1024)
+// %3 = splat(%1)
+// %4 = %3 + %2 == (pid * 1024) + tl.range(0,1024)
+// %5 = splat(arg0)
+// %6 = %5 + %4 = splat(arg0) + ((pid * 1024) + tl.range(0,1024))
+// %7 = %6 + %4 = splat(arg0) + ((pid * 1024) + tl.range(0,1024)) * 2
+// tt.load %7
+//
+// But for the tt.pointer_range=32, the tt.load's base-pointer and offset would
```

Comment:
"If with tt.pointer_range=32, tt.load's immediate pointer and offset would be ... But we try to track the the base pointer as far ahead as possible, so base pointer should be %5 and offset should be 2 x %4."

Reply:
rephrase the wording.